Course Screening

lsc0498 opened the pull request:
Adds the course review UI under /courses

lsc0498 committed:
feat: add new menu point for courses

lsc0498 committed:
feat: extract renderStatus from Queue to gain broader usability

lsc0498 committed:
feat: create course list page with tabs

Jonasdoubleyou commented:
we will continue this branch till the course review is working as we don't want to develop on master

Jonasdoubleyou committed:
Persistent routing without redirects

Jonasdoubleyou committed:
Prevent FullStory from crashing application

Jonasdoubleyou committed:
Make Dashboard main page and activate

Jonasdoubleyou committed:
Add Course Hook

Jonasdoubleyou committed:
Add Basic Course Table

Jonasdoubleyou committed:
fix things from review

Jonasdoubleyou committed:
Merge pull request #5 from corona-school/feat/course-api

Course API & Table that uses it

Jonasdoubleyou committed:
Disable Dashboard

Jonasdoubleyou committed:
Fix API context types

Jonasdoubleyou committed:
Merge commit '0691cbaca54cc462494b2ae72a12fe85be336821' into fix/persistent-login

Jonasdoubleyou committed:
Remove further unneccessary redirects

Jonasdoubleyou committed:
Merge pull request #4 from corona-school/fix/persistent-login

Persistent routing without redirects

Jonasdoubleyou committed:
Add CourseUpdate

Jonasdoubleyou committed:
Add Screening Comment

Jonasdoubleyou committed:
Correctly load table on first load

lsc0498 committed:
Add .css for course table

lsc0498 committed:
Create basic design for course details

lsc0498 committed:
Refactoring

lsc0498 committed:
Display instructor name in course details

lsc0498 committed:
Add functionality to add screener's comments

lsc0498 committed:
Close course overview by updating with course state undefined

This ensures that comments are saved also when the course is neither
accepted nor rejected.

Jonasdoubleyou committed:
Merge branch 'feat/course-review' of https://github.com/corona-school/web-screening-admin into feat/course-review

Jonasdoubleyou committed:
Replace POST with GET

Jonasdoubleyou committed:
Edit Course on Row click

lsc0498 committed:
Add EditMode for course desciptions

Jonasdoubleyou committed:
Merge branch 'feat/course-review' of https://github.com/corona-school/web-screening-admin into feat/course-review

Jonasdoubleyou committed:
Editable fields for name and category

Jonasdoubleyou committed:
Don't show unneccessary actions ("allow courses that are already allowed")

Commit:
Fixed width of course-title in edit mode

Commit:
Fixed Scrolling and added Tags display

Commit:
added tags to overview page

Jonasdoubleyou committed:
Remove unused code and hide courses form menu

Jonasdoubleyou committed:
fix course tags

Jonasdoubleyou committed:
Merge pull request #6 from corona-school/feat/course-review

Course Screening / Course Review UI

LeonErath commented:
Werde es heute nicht mehr schaffen und morgen vllt auch nicht. Also wenn ihr es eilig habt, dann testet einmal gut durch und mergt 👍🏻

Jonasdoubleyou commented:
@geroembser wie dringend ist das?

geroembser commented:
Es reicht, wenn es Sonntag deployed wird. Montag sollte es aber live sein. Wichtig ist nur, dass es nichts am vorhandnen Screening-Tool kaputt macht, muss es nicht super stable sein, weil das ja primär intern genutzt wird.
